src/app/ReadHandler.cpp

```diff
@@ -229,8 +225,8 @@ CHIP_ERROR ReadHandler::SendReportData(System::PacketBufferHandle && aPayload, b
         VerifyOrReturnLogError(mpExchangeCtx == nullptr, CHIP_ERROR_INCORRECT_STATE);
         VerifyOrReturnLogError(mSessionHandle, CHIP_ERROR_INCORRECT_STATE);
         mpExchangeCtx = mpExchangeMgr->NewContext(mSessionHandle.Get(), this);
-        mpExchangeCtx->SetResponseTimeout(kImMessageTimeout);
```

Comment:
Again, why are we removing this?

Reply:
per offline chat, we would blindly set it up in case we forge it even though we would no expect reply in some situation.